<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiGStream</class>
 <widget class="QMainWindow" name="UiGStream">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>625</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>GlobiStreams</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Streams</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="ui_mainLayout">
          <item>
           <layout class="QHBoxLayout" name="ui_singleStreamLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Stream URL :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ui_urlEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="ui_poll">
              <property name="text">
               <string>Poll</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ui_addToMonitored">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Monitor</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Monitored Streams :</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Events</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="ui_eventTableLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="ui_addEvent">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Add an Event</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Current Events :</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="ui_statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="ui_a_quit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="ui_a_netProperties"/>
    <addaction name="ui_a_mediaPlayer"/>
    <addaction name="ui_a_showOfflineStreams"/>
    <addaction name="ui_a_Non_Monitored_Streams"/>
    <addaction name="ui_a_gomTV"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="ui_a_about"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="ui_dockOutput">
   <property name="windowTitle">
    <string>Output Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="ui_dockChat">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Chat Browser</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="ui_actionChangeServer">
   <property name="text">
    <string>Change Server</string>
   </property>
  </action>
  <action name="ui_actionChangeMediaPlayer">
   <property name="text">
    <string>Change Media Player</string>
   </property>
  </action>
  <action name="ui_actionShowChat">
   <property name="text">
    <string>Show Chat</string>
   </property>
  </action>
  <action name="ui_a_quit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="ui_a_netProperties">
   <property name="text">
    <string>Network Properties</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="ui_a_mediaPlayer">
   <property name="text">
    <string>Media Player</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="ui_a_showOfflineStreams">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Offline Streams</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="ui_a_Non_Monitored_Streams">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Non-Monitored Streams</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="ui_a_gomTV">
   <property name="text">
    <string>GomTV account</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="ui_a_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="globistreams.qrc"/>
 </resources>
 <connections/>
</ui>
